packages/trader/src/Modules/MT5/Components/mt5-demo-account-display.jsx

```diff
@@ -42,15 +42,16 @@ const MT5DemoAccountDisplay = ({
             {(landing_companies?.mt_gaming_company?.financial || !is_logged_in) && (
                 <MT5AccountCard
                     has_mt5_account={has_mt5_account}
-                    icon={() => <Icon icon='IcMt5SyntheticPlatform' size={64} />}
+                    icon={() => <Icon icon='IcMt5Synthetic' size={32} />}
```

Comment:
`icon` needs to be a function?

Reply:
It does not need to be a function and also previously done by someone else. I will update it to just pass Icon and not a function. since i was redesigning the cards i tried not to impact the previous code as much as i could.